pkgs/development/libraries/gnutls/default.nix

```diff
@@ -57,8 +57,12 @@ stdenv.mkDerivation {
     "--disable-dependency-tracking"
     "--enable-fast-install"
     "--with-unbound-root-key-file=${dns-root-data}/root.key"
-  ] ++ lib.optional guileBindings
-    [ "--enable-guile" "--with-guile-site-dir=\${out}/share/guile/site" ];
+  ] ++ lib.optional guileBindings [
+    "--enable-guile"
+    "--with-guile-site-dir=\${out}/share/guile/site"
+    "--with-guile-site-ccache-dir=\${out}/share/guile/site"
```

Comment:
```suggestion
    "--with-guile-site-ccache-dir=\${out}/share/guile/site/site-ccache"
```

Reply:
I originally had it as `${out}/lib/guile/2.2/site-ccache`, based on the guile default on other systems being `/usr/lib/guile/2.2/site-ccache`. However, I changed it to `${out}/share/guile/site` because
1. That's what the only other package (`guile-xcb`) using this configure flag does.
2. There's no mention of a directory named `site-ccache` in `pkgs/development/interpreters/guile/setup-hook-2.2.sh`

Reply:
Does the fact your suggested `site-ccache` is a subdirectory of the `site` directory (which *is* added to the guile load paths), mean that it will work?

Reply:
Hmm. Since we seem to put it in `/share/guile` everywhere, let's do that and not `lib/guile`.

If it works without the `site-ccache` suffix, that's fine.
It's just a little bit more neat, if it's got its own subdirectory.
That's what they do upstream, too: https://gitlab.com/gnutls/gnutls/commit/499a021b968033057944492ee3074f0898aa4dba#d5b4de16d947214ec306bd57bed1bd23a939b5f9_23_27
And in Guix's "nixpkgs": http://git.savannah.gnu.org/cgit/guix.git/tree/gnu/packages/tls.scm#n200